src/test/java/net/phptravels/WebLogin.java

```diff
@@ -16,11 +16,47 @@ public class WebLogin implements IAbstractTest {
     @MethodOwner(owner = "qpsdemo")
     @TestPriority(Priority.P3)
     @TestLabel(name = "feature", value = {"web", "regression"})
-    public void testModelSpecs() {
+    public void testHotelsFillForm() {
         HomePage homePage = new HomePage(getDriver());
         homePage.open();
         Assert.assertTrue(homePage.isPageOpened(), "Home page is not opened");
```

Comment:
To make isPageOpened() work properly you need to specify in pages constructor set page opening strategy and setUILoadedMarker. I recommend using By Element strategy
Take a look here again on `Page opening strategy` section here https://zebrunner.github.io/carina/automation/web/